api/envoy/config/listener/v3/listener.proto

```diff
@@ -70,6 +72,7 @@ message Listener {
     option (udpa.annotations.versioning).previous_message_type =
         "envoy.api.v2.Listener.ConnectionBalanceConfig";
 
+    // [#extension: envoy.network.connection_balance.exact]
```

Comment:
I add this refer to STYLE.md, although it is not a "real" extension.

Same reason about changes to `source/extensions/extensions_build_config.bzl`.

As I asked in the slack(https://envoyproxy.slack.com/archives/C78HA81DH/p1651895235563049), it does not work under `source/common` :(

Reply:
I do not add `// [#protodoc-title: xx]` for this, so docs build fails. While we have docs for those parts, is still necessary?

@envoyproxy/envoy-maintainers if agree, I will add it to fix CI.